lumibot/strategies/strategy.py

```diff
@@ -3891,6 +3887,10 @@ def send_result_text_to_discord(self, returns_text, portfolio_value, cash):
         # Remove the extra spaces at the beginning of each line
         message = "\n".join(line.lstrip() for line in message.split("\n"))
 
+        # Add self.discord_account_summary_footer to the message
+        if hasattr(self, "discord_account_summary_footer") and self.discord_account_summary_footer is not None:
```

Comment:
you added this "discord_account_summary_footer" attribute to the __init__() function, so there is no need to do the "hasattr" check. It can be removed to improve readability.